common/funcfile.go

```diff
@@ -51,6 +51,15 @@ type Expects struct {
 	Config []inputVar `yaml:"config" json:"config"`
 }
 
+type paths struct {
+	From string `yaml:"from,omitempty" json:"from,omitempty"`
+	To string `yaml:"to,omitempty" json:"to,omitempty"`
+}
+
+type CopyResources struct {
```

Comment:
Since this struct is not used outside of this package, it can be made private by renaming to `copyResources`.